### ISSUE TITLE
Is there games compatibility list for AppleWin 

### ISSUE THREAD

**Issue by Darthfecker** [AppleWin/AppleWin#903]
Is there a games compatibility list for AppleWin? Just to make sure, because Champions of Krynn won't work on AppleWin.

**Comment by tomcw** [comment]
Can you provide a link (or attach a zip) of the non-working image(s).
Can you also provide some reproducible steps, so that I can see it failing.

In general most images work with AppleWin - currently there are none that I can think of that don't work (due to an emulation problem). If they don't work then it's _extremely likely_ to be an issue with the image, eg. copy-protection still present. This is especially true if the image is .DSK (or .DO/.PO) or .NIB... these days .WOZ is the image format of preference.

**Comment by tomcw** [comment]
I tried this from Asimov:
http://mirrors.apple2.org.za/ftp.apple.asimov.net/images/games/rpg/champions_of_krynn/ChampionsOfKrynn.zip

The KrynnMiniDocs.txt says:
>As of September 2000, this game will not run correctly on AppleWin.
>You can not reliably Save and Restore games or characters.

I booted it, and the game looks similar to these SSI AD&D titles: 'Curse of the Azure Bonds' and 'Pool of Radiance'. Support for these was fixed in #582, a few years ago.

Anyway, I was able to create 2 characters and save them to a save disk that I created. Then I could load the characters and form a party.

NB. To create a blank disk (from within AppleWin), just click either of drive-1 or drive-2 icons, and type the name of a file that doesn't exist, eg. KrynnSave.dsk. When you try to save to this, then the game will ask if you want to create a save disk.
